docs/en/latest/getting-started.md

```diff
@@ -125,6 +125,7 @@ First clone the [apisix-docker](https://github.com/apache/apisix-docker) reposit
 
 ```shell
 git clone https://github.com/apache/apisix-docker.git
+git checkout release/2.15
```

Comment:
Hi @darkSheep404, the Getting Started part will be replaced by https://docs.api7.ai/apisix/documentation. You can refer to these docs.

Reply:
hi @juzhiyuan thank you for provide new docs😃
but due to the current search result of apisix docs,i think this enhancement to this LTS version will do helps to learner like me too 😃
![image](https://user-images.githubusercontent.com/49020899/220802854-8b54308c-db62-47c3-9638-e684fe81f1bb.png)

Reply:
RIght. But I plan to submit another PR to update the contents totally. (Replace Getting Started in apisix.apache.org) Is it ok for you?

Reply:
That would be great if you did that😀